src/commands/until_front_of_line.yml

```diff
@@ -64,8 +68,14 @@ steps:
 
 
         load_oldest_running_build_num(){
-          curl -s $jobs_api_url_template > /tmp/jobstatus.json
-          if [ -f $TESTING_MOCK_RESPONSE ];then cat $TESTING_MOCK_RESPONSE > /tmp/jobstatus.json; fi
+          if [ -f $TESTING_MOCK_RESPONSE ];then 
+            echo "Using test mock response"
+            cat $TESTING_MOCK_RESPONSE > /tmp/jobstatus.json 
+          else
+            echo "Attempting to access CircleCI api. If the build process fails after this step, ensure your CIRCLECI_API_KEY is set."
+            curl -f -s $jobs_api_url_template > /tmp/jobstatus.json
```

Comment:
If the API key is needed when this curl happens it'll 404 and this command will fail the script. Hopefully that behavior makes sense. The api key isn't needed for public repos (unless there's a rate limit or something, but that's a bit out of scope as far as I'm concerned).